rhombus/gui/private/input.rhm

```diff
@@ -0,0 +1,77 @@
+#lang rhombus/static/and_meta
+import:
+  lib("racket/gui/easy.rkt")
+  rhombus/draw:
+    expose:
+      Color
+      Font
+  rhombus/draw/private/font!private._Font
+  rhombus/draw/private/symbol_map.symbol_map_annot
+  "label.rhm".Label.normal_control_font
+  "obs.rhm":
+    expose:
+      Obs
+      MaybeObs
+  "type.rhm":
+    expose:
+      LabelString
+      Margin
+      Size
+      Stretch
+  "view.rhm".View
+
+export:
+  Input
+
+
+
+
+class Input(hand, private _at_content :: Obs):
+  implements View
+  internal _Input
+  opaque
+  override property handle: hand
+
+  constructor (content :: MaybeObs.of(String) = "",
```

Comment:
I think this annotation isn’t quite right—see the note on `#:value->text`.

Reply:
Should the annotation be ommited? Like in `canvas.shm`?
```suggestion
  constructor (content,
```

Reply:
Omitting it is fine (conceptually, it’s `MaybeObs.of(Any)`, which is just `Any` anyway, but the docs can mention the former instead).